index.js

```diff
@@ -649,7 +658,7 @@ instance.prototype.action = function(action, extras) {
 	else if (id == 'exec') {
 		debug("Running path: '"+opt.path+"'");
 		exec(opt.path, {
-			timeout: 5
+			timeout: opt.timeout == null ? 5 : opt.timeout
```

Comment:
this should be `opt.timeout === undefined`

Reply:
Fixed. Thank you for catching that!

Reply:
I think the default timeout of 5 is too low. It should probably at least be something like 1000, if you ask me? Maybe even 5000 ?

Reply:
5000 sounds about right. Again, I was just leaving it at the previous value.